<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IfBlock_Canvas</class>
 <widget class="CBBaseBlock" name="IfBlock_Canvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>32</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IfBlock_Canvas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 89, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 89, 255);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="IF">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(181, 89, 255);</string>
        </property>
        <property name="text">
         <string>IF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="Param0">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DropDown">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="Param1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DO">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(181, 89, 255);</string>
        </property>
        <property name="text">
         <string>DO FOLLOWING</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CBBaseBlock</class>
   <extends>QWidget</extends>
   <header>cbbaseblock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
